README.md

```diff
@@ -1,4 +1,10 @@
-[![Build Status](https://travis-ci.org/aws/aws-xray-sdk-java.svg?branch=master)](https://travis-ci.org/aws/aws-xray-sdk-java)
+![Build Status](https://github.com/aws/aws-xray-sdk-java/actions/workflows/master-build.yml/badge.svg)
+
+### :mega: OpenTelemetry Java with AWS X-Ray
+
+AWS X-Ray recommends using AWS Distro for OpenTelemetry (ADOT) to instrument your application due to its wider range of features and instrumentations. See [Working with Java](https://docs.aws.amazon.com/xray/latest/devguide/xray-java.html) for more help with choosing between ADOT and X-Ray SDK.
```

Comment:
I think it’s useful to add this?

```suggestion
AWS X-Ray recommends using AWS Distro for OpenTelemetry (ADOT) to instrument your application **instead of this X-Ray SDK** due to its wider range of features and instrumentations. See the [AWS X-Ray docs on Working with Java](https://docs.aws.amazon.com/xray/latest/devguide/xray-java.html) for more help with choosing between ADOT and X-Ray SDK.
```